eZ/Publish/API/Repository/Tests/ContentServiceTest.php

```diff
@@ -2163,6 +2163,90 @@ public function testUpdateContentMetadataThrowsInvalidArgumentExceptionOnNoMetad
         $contentService->updateContentMetadata($contentInfo, $contentMetadataUpdateStruct);
     }
 
+    /**
+     * @covers \eZ\Publish\API\Repository\ContentService::updateContentMetadata
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\ForbiddenException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    public function testUpdateContentAlwaysAvailable()
+    {
+        $repository = $this->getRepository();
+        $contentService = $repository->getContentService();
+
+        $folder = $this->createFolder(['eng-GB' => 'Folder'], 2);
+
+        $contentMetadataUpdate = $contentService->newContentMetadataUpdateStruct();
+        $contentMetadataUpdate->alwaysAvailable = !$folder->contentInfo->alwaysAvailable;
+        $contentService->updateContentMetadata($folder->contentInfo, $contentMetadataUpdate);
+
+        $reloadedFolder = $contentService->loadContent($folder->contentInfo->id);
```

Comment:
```suggestion
        $reloadedFolder = $contentService->loadContent($folder->id);
```

or should we load from ContentInfo?